Examples/restcomm-olympus-nopod/restcomm-olympus/MainTableViewController.mm

```diff
@@ -678,4 +677,14 @@ - (void)reloadData{
     [self.tableView reloadData];
 }
 
+#pragma mark - Helper
+
+- (UIButton *)getBugReportButton{
+    UIButton *bugButton =  [UIButton buttonWithType:UIButtonTypeCustom];
+    [bugButton setImage:[UIImage imageNamed:@"bug-grey-icon-25x25.png"] forState:UIControlStateNormal];
```

Comment:
Unfortunately this icon seems a bit huge and thick. Let me involve @ipsilantide so that we can brainstorm on this. Please leave this one for now until we reach consensus on this

Reply:
Ok.